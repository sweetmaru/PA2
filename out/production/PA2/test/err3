1
remove 0
end
