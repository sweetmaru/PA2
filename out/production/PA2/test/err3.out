Please enter initial arrangement of books followed by newline:
[1] 0 0
Type pick <index> or put <height> followed by newline. Type end to exit.
ERROR: Invalid command. Valid commands are pick, put, or end.
Exiting Program.
